microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshots.md

```diff
@@ -317,6 +296,74 @@ To read more about DOM leaks and memory analysis fundamentals, check out [Findin
 <!--[narrow down causes of memory leaks](../profile/memory-problems/memory-diagnosis#narrow-down-causes-of-memory-leaks) -->
 
 
+<!-- ====================================================================== -->
+## Analyze the impact of closures on memory
+
+To analyze the impact of closures on memory, try out this example:
+
+1. Open the [Eval is evil](https://microsoftedge.github.io/Demos/devtools-memory-heap-snapshot/example-07.html) demo webpage in a new window or tab.
+
+1. Record a first heap snapshot.
+
+1. In the rendered webpage, click the **Closures with eval** button.
+
+1. Record a second heap snapshot.
+
+   In the sidebar, the number below the second snapshot should be larger than the number below the first snapshot. This indicates that more memory is being used by the webpage after clicking the **Closures with eval** button.
+
+1. In the second heap snapshot, change the view to **Comparison**, comparing it to the first one.
```

Comment:
```suggestion
1. In the second heap snapshot, change the view to **Comparison**, comparing the view or snapshot to the first view or snapshot.
```
manually apply this sketchy wording change

"it" ... "one" - i'm lost.  too many indirection wildcard pointer words.  Speak directly.

the phrase "comparing it to the first one" cannot be made sense of, w/o labor of analysis to resolve indirection pointer words